src/contracts/pods/EigenPod.sol

```diff
@@ -437,7 +431,6 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         bytes calldata validatorFieldsProof,
```

Comment:
in line 402, we are calling this the `latestBlockHeaderRoot`. seems like we should call this `blockHeaderRoot`

Reply:
I believe I made the same comment elsewhere. Per @Sidu28 this is literally called the `latestBlockHeaderRoot` in the spec. I think at minimum we need to add some comments to clarify; if you have ideas where, please make a commit!

Reply:
This was where the proof was broken

Reply:
> This was where the proof was broken

please expand on this idea / I guess file an issue for @Sidu28 for this